doc/src/site/sphinx/deployment/kubernetes.rst

```diff
@@ -246,3 +246,120 @@ Dynamic allocation must be disabled in Spark.
             Rscript --default-packages=methods,utils batch.R
 
         Note: In case of RSparkling, SparklyR automatically sets the Spark deployment mode and it is not possible to specify it.
+
+External Backend
+~~~~~~~~~~~~~~~~
+
+Sparkling Water External backend can be also used in Kubernetes. First, we need to start external H2O backend on
+Kubernetes. This can be achieved by the following steps:
+
+1. Create Headless Service for H2O node discovery
```

Comment:
Isn't it better to add just a link to h2o documentation and tell users to use a different image? I'm just afraid that this configuration will go out of sync with the H2O ones.

Reply:
Agree, will just point to h2o documentation